tests/foreman/ui/test_computeresource_gce.py

```diff
@@ -57,22 +56,23 @@ def test_positive_default_end_to_end_with_custom_profile(
 
     :CaseImportance: Critical
     """
-
     cr_name = gen_string('alpha')
     new_cr_name = gen_string('alpha')
     cr_description = gen_string('alpha')
-    new_org = entities.Organization().create()
-    new_loc = entities.Location().create()
+    new_org = sat_gce.api.Organization().create()
+    new_loc = sat_gce.api.Location().create()
     json_key = json.dumps(gce_cert, indent=2)
-    http_proxy = entities.HTTPProxy(
+    http_proxy = sat_gce.api.HTTPProxy(
         name=gen_string('alpha', 15),
         url=settings.http_proxy.auth_proxy_url,
         username=settings.http_proxy.username,
         password=settings.http_proxy.password,
-        organization=[module_org.id],
-        location=[module_location.id],
+        organization=[sat_gce_org.id],
+        location=[sat_gce_loc.id],
     ).create()
     with session:
```

Comment:
If we think to run any of this tests against puppet_sat, or we can be explicicit by using ui_session 
```suggestion
    with sat_gce.ui_session() as session:
```

Reply:
I think the session is generic and should not be doing anything with the sat_object being used @omkarkhatavkar @shweta83  ?